underlay/src/main/proto/criteriaselector/configschema/multi_attribute.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package tanagra.configschema;
+
+option go_package = "github.com/DataBiosphere/tanagra/criteriaselector/configschemapb";
+option java_package = "bio.terra.tanagra.proto.criteriaselector.configschema";
+option java_outer_classname = "CFMultiAttribute";
+
+import "criteriaselector/value_config.proto";
+
+message MultiAttribute {
+  string entity = 1;
```

Comment:
We can do that in a separate PR. I want this one to stay as much a direct conversion as possible to avoid introducing new issues and not make this already large PR any larger.

Reply:
OK, sounds good. Doesn't have to be here, but this would be very helpful to have before hooking up the backend config.